### PR TITLE
:hammer: volume upgrade test

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #PTX-4731

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by lsrinivas-pure** [comment]
tp-dev-test : https://jenkins.portworx.dev/job/Torpedo/job/tp-dev-test/988/console

File: drivers/volume/portworx/portworx.go [hunk 1]
```diff
@@ -2974,6 +2982,44 @@ func (d *portworx) ToggleCallHome(n node.Node, enabled bool) error {
...
+	stc.Spec.Image = imageName
+	updatedStc, err := pxOperator.UpdateStorageCluster(stc)
+	if err != nil {
+		er := fmt.Errorf("error upgrading Storage Cluster [%v], Namespace: [%v], Err: %v", pxOps.Items[0].Name, pxOps.Items[0].Namespace, err.Error())
+		logrus.Error(er.Error())
+		return er
+
+	}
+	logrus.Infof("Storage Cluster Image updated to [%v]", imageName)
+	logrus.Info(updatedStc)
```
Comment (Rohit-PX) [inline drivers/volume/portworx/portworx.go:3018]: I don't think we need to print the entire storage cluster here.

**Review by Rohit-PX — APPROVED** [review]
Looks good except the changes pointed out.

File: drivers/volume/portworx/portworx.go [hunk 2]
```diff
@@ -2974,6 +2982,44 @@ func (d *portworx) ToggleCallHome(n node.Node, enabled bool) error {
 	return nil
 }
 
+func (d *portworx) IsOpearatorBasedInstall() (bool, error) {
```
[the remark this answers is not in the document]
Comment (lsrinivas-pure) [reply drivers/volume/portworx/portworx.go]: fixed

File: drivers/volume/portworx/portworx.go (hunk 1 above)
Comment (lsrinivas-pure) [reply drivers/volume/portworx/portworx.go]: fixed

File: drivers/volume/volume.go [hunk 3]
```diff
@@ -225,6 +228,12 @@ type Driver interface {
 	// UpdateSharedv4FailoverStrategyUsingPxctl updates the sharedv4 failover strategy using pxctl
 	UpdateSharedv4FailoverStrategyUsingPxctl(volumeName string, strategy api.Sharedv4FailoverStrategy_Value) error
 
+	//IsOpearatorBasedInstall returns if px is operator based
+	IsOpearatorBasedInstall() (bool, error)
```
[the remark this answers is not in the document]
Comment (lsrinivas-pure) [reply drivers/volume/volume.go]: fixed

**Comment by vilasd** [comment]
Why this volume upgrade

**Pull request merged**